.travis.yml

```diff
@@ -20,7 +20,7 @@ deploy:
     skip_cleanup: true
     script: lein do git-info-edn, uberjar && mkdir -p target/oc && cp target/*-standalone.jar target/oc/ && ./deploy-to-openshift.sh $TRAVIS_BRANCH $TRAVIS_PULL_REQUEST
     on:
-      all_branches: true
+      branch: dev
```

Comment:
Doesn't this change prevent merges to master from being deployed to OpenShift?

Reply:
No, it prevents all branches to deploy on OpenShift (ssf-dev project), for example the `mbeans` branch. The `master`  branch deployment configuration is defined in the block below - see https://github.com/symphonyoss/bot-github-chatops/blob/master/.travis.yml#L26

Without this change, all Travis PR builds would fail, since travis tries to run the `./deploy-to-openshift.sh` script, which fails because it only knows how to deploy `master` and `dev` branches.

Reply:
Where is deploy-to-openshift.sh called in the next `provider` block?  I don't see it...

Also, shouldn't deploy-to-openshift.sh be more defensive, and silently exit if it doesn't recognise the branch name?  That would be a more robust approach.

Reply:
Added condition on travis build - https://github.com/symphonyoss/bot-github-chatops/commit/72d92562f9319fa6165990eb1b8f928169bf4a5c

Re. deploy-to-openshift.sh, I'm still checking how to make https://github.com/symphonyoss/contrib-toolbox/blob/master/scripts/oc-deploy.sh more defensive